api/handler/acl.go

```diff
@@ -167,6 +167,8 @@ func (h *handler) GetBucketACLHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	h.log.Debug("target details", zap.Stringer("cid", bktInfo.CID))
```

Comment:
Actually we should log the bucket name also, otherwise we can miss mapping because of concurrent logs order.

Also, maybe we should use `zap.String("cid", bktInfo.CID.EncodeToString())`?